### PR TITLE
Re-enable flakytest GAWB-3138 for investigation

### PR DESCRIPTION
**Pull request by jmthibault79**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
Depends on https://github.com/broadinstitute/firecloud-automated-testing/pull/9

Running this test locally manually did not cause any errors.  For Google to investigate more thoroughly, they need a version of the test which doesn't delete the project after.  Enter GPAlloc!  I copied the example at #1200 to enable that.

Also shortened the wait time and added some debug prints.  We can remove those later.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala [hunk 1]
```diff
@@ -9,18 +11,22 @@ import org.broadinstitute.dsde.workbench.dao.Google.googleBigQueryDAO
...
-import org.scalatest.time.{Minutes, Second, Seconds, Span}
+import org.scalatest.time.{Minutes, Seconds, Span}
 import org.scalatest.{FreeSpec, Matchers}
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 
 class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with Eventually
   with BillingFixtures {
   implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(5, Seconds)))
 
+  def nowPrint(text: String): Unit = {
```
Comment (jmthibault79) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala:22]: Can remove these debug prints later ... or now :)

**Comment by breilly2** [comment]
Is your intent to merge this? Now that you have a branch, you can gather data by asking jenkins to run a bunch of test pipelines. Don't even need the PR.

**Comment by jmthibault79** [comment]
I was unable to produce an error by running locally or asking Jenkins to run this.  So I would like to merge this and run it as part of our normal process.

**Comment by jmthibault79** [comment]
jenkins retest

**Review by breilly2 — APPROVED** [review]
Sounds like you have evidence to suggest that, for whatever reason, the test is no longer flaky. 👍  And the logging will help diagnose further if it does fail.

**Comment by jmthibault79** [comment]
jenkins retest

**Pull request merged**